rapidtide/correlate.py

```diff
@@ -18,36 +18,27 @@
 # $Author: frederic $
 # $Date: 2016/07/12 13:50:29 $
 # $Id: tide_funcs.py,v 1.4 2016/07/12 13:50:29 frederic Exp $
-#
-from __future__ import print_function, division
```

Comment:
Removed since 2.7 is no longer supported.

Reply:
I've gone ahead and removed that line from all files now.